src/alfrescoApiClient.js

```diff
@@ -205,6 +205,11 @@ class AlfrescoApiClient extends ApiClient {
         // set header parameters
         request.set(this.defaultHeaders).set(this.normalizeParams(headerParams));
 
+        // set the proxy user name if present
+        if (!!proxyUserName) {
```

Comment:
`proxyUserName` needs to be set on a per-request basis, so we can't just set it once per client.  The header is set per-client (hence initializing it in the constructor), but the value is per-request.